### PR TITLE
fx layout legacy directives

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Migrieren von flex-layout](https://trello.com/c/14u8jeFr/1159-migrieren-von-flex-layout)

**Review by jdroenner — CHANGES_REQUESTED** [review]

File: projects/core/src/lib/time/small-time-interaction/small-time-interaction.component.scss [hunk 1]
```diff
@@ -2,11 +2,7 @@
     padding: 2px 2px;
 }
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply projects/core/src/lib/time/small-time-interaction/small-time-interaction.component.scss:4]: [98df854](https://github.com/geo-engine/geoengine-ui/pull/596/commits/98df854f4b943d70475aea6e5095197958a37b55)

File: projects/core/src/lib/layer-collections/layer-collection-layer/layer-collection-layer.component.html [hunk 2]
```diff
@@ -24,7 +24,7 @@ <h4 class="header-text-line">{{ layer?.name }}</h4>
 </ng-container>
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply projects/core/src/lib/layer-collections/layer-collection-layer/layer-collection-layer.component.html:25]: [628daea](https://github.com/geo-engine/geoengine-ui/pull/596/commits/628daea827c548fa65ee4b708f04916d73f9f189)

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Comment by jdroenner** [comment]
bors retry

**Pull request merged**